example/mqtt/test_mqtt.tavern.yaml

```diff
@@ -0,0 +1,112 @@
+---
+
+test_name: Test mqtt message response
+
+includes:
+  - !include common.yaml
+
+mqtt: &mqtt_spec
+  # tls:
+  #   enable: true
+  connect:
+    host: localhost
+    port: 9001
+    timeout: 3
+  client:
+    transport: websockets
+    client_id: tavern-tester
+
+stages:
+  - name: step 1 - ping/pong
+    mqtt_publish:
+      topic: /device/123/ping
+      payload: ping
+    mqtt_response:
+      topic: /device/123/pong
+      payload: pong
```

Comment:
What happens if more than one message is received on this topic, but the first N messages do not meet the validation?

Reply:
They are ignored until the specified timeout has passed, at which point it fails the test